src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -0,0 +1,145 @@
+  */
+case class DplaSourceResource(
+                               alternateTitle: ZeroToMany[String] = Seq(),
+                               collection: ZeroToMany[DcmiTypeCollection] = Seq(),
+                               contributor: ZeroToMany[EdmAgent] = Seq(),
+                               creator: ZeroToMany[EdmAgent] = Seq(),
+                               date: ZeroToMany[EdmTimeSpan] = Seq(),
+                               description: ZeroToMany[String] = Seq(),
+                               extent: ZeroToMany[String] = Seq(),
+                               format: ZeroToMany[String] = Seq(),
+                               genre: ZeroToMany[SkosConcept] = Seq(),
+                               identifier: ZeroToMany[String] = Seq(),
+                               language: ZeroToMany[SkosConcept] = Seq(),
+                               place: ZeroToMany[EdmPlace] = Seq(), //specified as dpla:Place in the spec
+                               publisher: ZeroToMany[EdmAgent] = Seq(),
+                               relation: ZeroToMany[LiteralOrUri] = Seq(),
+                               replacedBy: ZeroToMany[String] = Seq(),
+                               replaces: ZeroToMany[String] = Seq(),
+                               rights: AtLeastOne[String] = Seq(),
+                               rightsHolder: ZeroToMany[EdmAgent] = Seq(),
+                               subject: ZeroToMany[SkosConcept] = Seq(),
+                               temporal: ZeroToMany[EdmTimeSpan] = Seq(),
+                               title: AtLeastOne[String] = Seq(),
+                               `type`: ZeroToMany[String] = Seq() //should be URIs?
+                             )
+
+/**
+  * Contains the attributes of the digital representation of the web resource, not the source resource.
+  *
+  * Used both to talk about the item in situ on the provider's site, along with other entities on the Web.
+  */
+case class EdmWebResource(
+                           uri: ExactlyOne[URI],
+                           fileFormat: ZeroToMany[String] = Seq(),
+                           dcRights: ZeroToMany[String] = Seq(),
+                           edmRights: ZeroToOne[String] = None //todo should be a URI?
+                         )
+
+/**
+  * The aggregation of attributes that apply to the described resource as a whole,
+  * grouped from edm:WebResource and dpla:SourceResource.
+  *
+  */
+
+case class OreAggregation(
+                           uri: ExactlyOne[URI], //uri of the record on our site
+                           dataProvider: ExactlyOne[EdmAgent],
+                           originalRecord: ExactlyOne[String], //map v4 specifies this as a ref, but that's LDP maybe?
+                           hasView: ZeroToMany[EdmWebResource] = Seq(),
+                           intermediateProvider: ZeroToOne[EdmAgent] = None,
+                           `object`: ZeroToOne[EdmWebResource] = None, //full item page
```

Comment:
I'm now wondering if I just prefix these variables with an underscore, if that'll make the warning go away, and be "close enough."